docs/serving/using-auto-tls.md

```diff
@@ -0,0 +1,206 @@
+---
```

Comment:
@ZhiminXiang im still editing this file but I think you can begin reviewing the other pages and acting on some of my requests while i wrap this page up.

Reply:
edited, PTAL

Reply:
Thanks for updating the doc! It looks good!

Generally, one step is missing. So the general installation steps are:
1. Install cert-manager
2. Configure cert-manager. Specifically for Google Cloud DNS, we need to 1) [create DNS service account and upload the credential of DNS servers](https://github.com/knative/docs/blob/master/docs/serving/using-cert-manager-on-gcp.md#creating-a-cloud-dns-service-account) 2) [Create `ClusterIssuer`](https://github.com/knative/docs/blob/master/docs/serving/using-cert-manager-on-gcp.md#specifying-a-certificate-issuer)
3. Configure Knative, which includes configure `config-certmanager` ConfigMap and configure `config-network` ConfigMap.

I think we miss step 2 in this doc.

Reply:
Is step #2 specific to only GCP? I tried to make this step generic so that it applies to all platforms.

Perhaps I can mention the Cloud DNS specific steps as part of the Example, along with the pointer to the full Cloud DNS doc (https://github.com/knative/docs/pull/1281/files#diff-1ac0a9c578d517c34bf1c688503c8733R63)

Reply:
Step 2 is not specific to only GCP. Users need to configure DNS01 challenge provider for cert-manager based on their platform. Here is the [instruction](https://docs.cert-manager.io/en/latest/tasks/acme/configuring-dns01/index.html#supported-dns01-providers).

But in step 2, we can also provide a detailed instructions for Google Cloud DNS users, which is what we already have https://github.com/knative/docs/blob/master/docs/serving/using-cert-manager-on-gcp.md#creating-a-cloud-dns-service-account and https://github.com/knative/docs/blob/master/docs/serving/using-cert-manager-on-gcp.md#specifying-a-certificate-issuer

Reply:
ok ill try to add a generic step

Reply:
I took another look at this PR. Actually step 2 is already covered in the Prerequisite [installing-cert-manager](https://github.com/knative/docs/pull/1281/files#diff-80544c14efa434e63e3f49b606cbe6adR72). So we don't need to add it into this doc.